docs/content/en/docs/yaml-crd-ref/metricsprovider.md

```diff
@@ -1,5 +1,125 @@
 ---
 title: KeptnMetricsProvider
-description: Define all workloads and checks associated with an application
+description: Define data provider used for metrics and evaluations
 weight: 55
 ---
+
+`KeptnMetricsProvider` defines an instance of the data provider
+(such as Prometheus, Dynatrace, or Datadog)
+that is used by the [KeptnMetric](metric.md)
+and [KeptnEvaluationDefinition](evaluationdefinition.md) CRDs.
```

Comment:
You can leave out the mention of the `KeptnEvaluationDefinition` CRD here, as they now always refer to the `KeptnMetric` and hence do not have to be aware of the providers

Reply:
So what is now the difference between a Keptn Evaluation and a Keptn metric if even the KeptnEvaluationDefinition just refers to KeptnMetric?  If I understand that, I'll know how to straighten this out.

Reply:
The difference KeptnMetric basically just provides the value of the metric, while the KeptnEvaluation allows you to define success criteria for the referenced KeptnMetric in order to pass the pre/post evaluation stage

Reply:
done